api/AudioBufferSourceNode.json

```diff
@@ -445,15 +443,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode/start",
           "support": {
             "chrome": {
-              "version_added": "14",
-              "version_removed": "57"
```

Comment:
The 57 here was probably added because `start` went away from `AudioBufferSourceNode.prototype`, see https://github.com/mdn/browser-compat-data/issues/6656#issuecomment-688998331.

Reply:
It's quite possible -- the data came from Confluence, it appears (see #1346).